ansible/roles/mikegleasonjr.firewall/tasks/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+
+- name: Install iptables
```

Comment:
It’s helpful to keep upstream roles unmodified. Can we move this into `bootstrap` (or perhaps see if the author would take it upstream)?